package.json

```diff
@@ -73,7 +73,8 @@
   "overrides": {
     "@glimmer/validator": "0.84.3",
     "@glimmer/util": "0.84.3",
-    "ember-source": "$ember-source"
+    "ember-source": "$ember-source",
+    "ember-cli-htmlbars": "^6.3.0"
```

Comment:
This was added to fix an issue with ember canary, but that issue will be fixed upstream so we can remove the override.